spec/models/plugins/fundraiser_spec.rb

```diff
@@ -31,4 +31,15 @@
     expect( fundraiser.liquid_data[:donation_bands]).to eq "null"
   end
 
+  it 'serializes a named donation band' do
+    allow(PaymentProcessor::Currency).to receive(:convert)
+    _ = create :donation_band # We create this because it would be the default if the named band wasn't found.
```

Comment:
just fyi, there's no need to assign when using FG's `create`, if you're not going to use the product.